app/models/featured_content_feature.rb

```diff
@@ -1,3 +1,5 @@
 class FeaturedContentFeature < ApplicationRecord
   belongs_to :featured_content
+
+  validates :fromdate, :todate, presence: true
```

Comment:
previously we had this check in controller code. I've moved it to model's validation